provision/provision.sh

```diff
@@ -1,19 +1,23 @@
 #!/bin/bash
+echo "Installing yarn..."
+curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
+echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
+
 echo "Updating apt repositories..."
 apt-get update
 
 
 echo "Installing base packages..."
 PACKAGES="build-essential zsh git vim-nox tree htop libjpeg-dev libfreetype6-dev graphviz gettext"
 PACKAGES="$PACKAGES python3 python3-setuptools python3-pip python3-dev"
-PACKAGES="$PACKAGES postgresql-9.3 postgresql-server-dev-9.3"
-PACKAGES="$PACKAGES nginx"
+PACKAGES="$PACKAGES postgresql-10 postgresql-server-dev-10"
```

Comment:
¿Y si empezamos a usar Postgresql 11? 

https://www.citusdata.com/blog/2018/09/11/postgresql-11-just-in-time/ solo por tomar en cuenta el performance